six/modules/c++/six.sicd/include/six/sicd/Utilities.h

```diff
@@ -337,6 +337,49 @@ class Utilities
      * \return mock ComplexData object
      */
     static std::auto_ptr<ComplexData> createFakeComplexData();
+
```

Comment:
I think there is enough code here, and it's specific enough to area plane derivation, that all this should get its own class in its own header/source file